test/test-postgrator.js

```diff
@@ -147,20 +151,20 @@ buildTestsForConfig({
 	migrationDirectory: __dirname + '/migrations',
 	driver: 'pg.js',
 	host: 'localhost',
-	database: 'postgrator',
-	username: 'postgrator',
-	password: 'postgrator'
+	database: DB_NAME,
+	username: DB_USER,
+	password: DB_PASS,
 });
 
 
-buildTestsForConfig({
-	migrationDirectory: __dirname + '/migrations',
-	driver: 'mysql',
-	host: 'localhost',
-	database: 'test',
-	username: 'root',
-	password: ''
-});
+// buildTestsForConfig({
```

Comment:
Why comment these tests out? Can we add a travis config for running these tests to this?